src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java

```diff
@@ -130,25 +136,44 @@
     /** Default maximum number of columns in a line. */
     private static final int DEFAULT_MAX_COLUMNS = 80;
 
-    /** Patterns matching package, import, and import static statements. */
-    private static final Pattern IGNORE_PATTERN = Pattern.compile("^(package|import) .*");
+    /** Pattern matching import and import static statements. */
+    private static final Pattern IMPORT_PATTERN = Pattern.compile("^import .*");
+
+    /** Pattern matching package statements. */
+    private static final Pattern PACKAGE_PATTERN = Pattern.compile("^package .*");
```

Comment:
whitespaces is allowed before import/package?

Reply:
Yes it is allowed but I didn't include it in regex because I have used `find()` and not `matches()`.

Reply:
Also I have one more concern, consider this case-
```
import com.puppycrawl.tools.checkstyle.checks 
        .javadoc.utils.BlockTagUtil;
```
If we are considering the default config, assume that second line exceeds length of 80, it will give a violation because the check is based on regex and not on tokens, though this is a very unusual case but still I would mention it in documentation.

Reply:
Also I tried to use `assertWithMessage` instead of `verify` in the new tests but I am having little trouble doing so, I get NPE whenever I try to call the `processFiltered` method, I debugged it and found the `configuration` to be null. How to use truth in these unit test?

Reply:
done